hri_safety_sense/src/JoystickHandler.h

```diff
@@ -30,7 +30,7 @@ namespace hri_safety_sense {
 	 */
 	class JoystickHandler : public MsgHandler {
 	   public:
-		  JoystickHandler();
+		  JoystickHandler(const std::string &frameId="/srcs");
```

Comment:
We may as well remove the default from this argument; we aren't using it anyway since we always pass it in during the constructor.  This also ensures we don't have the default in two different places.

Reply:
Done in fc1be8f